app/src/main/java/org/mozilla/focus/fragment/FirstrunFragment.java

```diff
@@ -142,7 +142,7 @@ private void finishFirstrun() {
 
         fragmentManager
                 .beginTransaction()
-                .remove(this)
+                .replace(R.id.container, UrlInputFragment.createWithoutSession(), UrlInputFragment.FRAGMENT_TAG)
```

Comment:
This will break the flow if the app was opened with an Intent, e.g. ACTION_VIEW or a custom tab. Maybe that's okay?